lib/item_repository.rb

```diff
@@ -0,0 +1,44 @@
+require_relative "./item"
+
+class ItemRepository
+  def all
+    items = []
+    sql = "SELECT * FROM items;"
+    result = DatabaseConnection.exec_params(sql, [])
+
+    result.each do |record|
+      item = Item.new
+      item.id = record["id"]
+      item.item_name = record["item_name"]
+      item.price = record["price"]
+      item.quantity = record["quantity"]
+      item.order_id = record["order_id"]
+
+      items << item
+    end
+    items
```

Comment:
I like how you used newlines and had a very readable code, which I didn't, because rubocop was detecting too many lines as an offence